lib/active_fedora/file.rb

```diff
@@ -139,6 +145,20 @@ def metadata?
       false
     end
 
+    def external_content?
+      external_url.present?
+    end
+
+    def external_url
+      return nil unless mime_type.start_with? "message/external-body"
+      url = mime_type.split(';').at(2)
+      url.nil? ? nil : url[/\"(.*?)\"/, 1]
```

Comment:
I thought one of the HTTP clients would have this (and they must internally), but I couldn't find anything.  I did find a gem called content-type that will do this: https://rubygems.org/gems/content-type

```
ContentType.parse("message/external-body; access-type=\"URL\"; URL=\"http://example.org/1\"").parameters['url']
```